plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -0,0 +1,451 @@
+const {
+	test,
+	expect
+} = require('@playwright/test');
+const {
+	customer
+} = require('../../data');
+
+/**
+ * Tests for the WooCommerce Customers API.
+ *
+ * @group api
+ * @group customers
+ *
+ */
+test.describe('Customers API tests: CRUD', () => {
+	let customerId;
+
+	test.describe('Retrieve after env setup', () => {
+
+		// when the environment is created, we have an admin user and 
```

Comment:
Minor suggestion: It might be worth linking to the [GH documentation](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/e2e-pw#woocommerce-playwright-end-to-end-tests) for the environment so there's an easy reference back to it so folks know which environment is being referenced to here.

An aside, we'll also want to check into how this might work in a dynamic situation where a user is pointing these tests to their own test environment they have configured.

```suggestion
		// when the environment is created (https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/e2e-pw#woocommerce-playwright-end-to-end-tests), we have an admin user and 
```

Reply:
Agreed! These tests will only work in the instances we are setting the the test environment up and adding the subscriber customer/user through the CLI script.
There will possibly be other occasions that need to be considered where a user is pointing these tests to their own test environment they have configured.

Reply:
Thank you for making this change!